lms/static/scripts/frontend_apps/components/GroupConfigSelector.tsx

```diff
@@ -75,11 +83,8 @@ function GroupSelect({ busy, groupSets, onInput, selectedGroupSetId }) {
 
 /**
  * ErrorModal shown when the fetched list of course group sets is empty
- *
- * @param {object} props
- *   @param {() => void} props.onCancel
  */
-function NoGroupsError({ onCancel }) {
+function NoGroupsError({ onCancel }: { onCancel: () => void }) {
```

Comment:
Since this is a so simple object shape and the component is internal to this module, I have inlined the type, but I'm open to extract it to its own type definition.

Reply:
I think we should stick with the pattern of exporting props types here if that's OK (so, defining the type even though it only has one property)!

Reply:
Yeah, this is fine because this component is private to this module.